README.md

```diff
@@ -28,6 +28,7 @@ The `consul-register` option is keyval based, and it takes the following mendato
 
 Optional key options:
 
+* `token` : the consul token to use.
```

Comment:
please sort the option alphabetically, just like you rightfully did on the rest of the code :)